test/JDBC/upgrade/13_4/preparation/BABEL-3144-vu-prepare.sql

```diff
@@ -0,0 +1,110 @@
+-- max values causing overflow in sum (BIGINT,INT) 
```

Comment:
We returned different return types for sum aggregates in these versions, the way upgrade framework picks up the test is if its present in schedule  of the base version (both verify and prepare ,before and after the upgrade) so since after upgrade the dependant views from these versions still continue to return different return types we need different prepare and verify script